src/version/ContractVersionBase.sol

```diff
@@ -8,6 +8,6 @@ import {IVersionedContract} from "../interfaces/IVersionedContract.sol";
 contract ContractVersionBase is IVersionedContract {
     /// @notice The version of the contract
     function contractVersion() external pure override returns (string memory) {
-        return "1.3.0";
+        return "1.3.1";
```

Comment:
i wonder if this should always have to match the package version - what if we are just publishing js changes to npm?